flink-libraries/flink-gelly/src/main/java/org/apache/flink/graph/library/linkanalysis/HITS.java

```diff
@@ -168,7 +168,6 @@ protected void mergeConfiguration(GraphAlgorithmWrappingBase other) {
 				.setParallelism(parallelism)
 				.name("Square")
 			.reduce(new Sum())
-			.setCombineHint(CombineHint.HASH)
```

Comment:
I had updated the commit message which was not reflected in the pull request message. I've updated the latter.